Jenkinsfile

```diff
@@ -631,6 +636,77 @@ pipeline {
       }
     }
 
+    stage('Deploy Preview') {
+      when {
+        branch "${REFERENCE_BRANCH}"
+      }
+      steps {
+        setGitHubBuildStatus('nuxeo/preview/deploy', 'Deploy nuxeo Preview', 'PENDING')
+        container('maven') {
+          dir('docker/nuxeo') {
+            echo """
+            ----------------------------------------
+            Build Nuxeo preview image
```

Comment:
Images are already built by another stage "Build Docker images", no need to build them again...